src/games/snake/snake_part.py

```diff
@@ -0,0 +1,24 @@
+import pygame
+from src.resources.snake_game_resources import SnakeGameResources
+
+
+class SnakePart(pygame.sprite.Sprite):
+    def __init__(self, belongs_to, board, pos_center = None):
```

Comment:
You shouldn't use spaces in default parameter in python:
```suggestion
    def __init__(self, belongs_to, board, pos_center=None):
```